tools/vf-component-library/src/site/design-tokens/theming.njk

```diff
@@ -0,0 +1,37 @@
+---
+is_index: true
+title: Design Tokens
+subtitle: Theming
+date: 2018-10-19 12:24:50
+section: components
+tags: sections
+templateEngineOverride: njk
+layout: layouts/tokens.njk
+hideSubPosts: true
+---
+
+<nav class="vf-navigation vf-navigation--main">
```

Comment:
Should move the navigation to `_includes/layouts/tokens.njk` or some other include?

Reply:
I would but I think it will break because of it thinking in relative paths and stuff?

I could see how we can 'generate it' using 11ty -- which is what I'd really want to do here.

Reply:
Think it could work. If you do `href="{{ '/design-tokens/typography' | url}}"` _should_ work fine

Reply:
I'll take a look :)

Reply:
updated and resolved

Reply:
![navigation is fun](https://media.tenor.co/images/c5eb515b7a7e200545eb071aeace7d0f/tenor.gif)